department-of-veterans-affairs/caseflow
# Add FNOD badge to Queue Case List table

Resolves [CASEFLOW-47](https://vajira.max.gov/browse/CASEFLOW-47)

### Description
Add FNOD Badge to Queue Case List table
FNOD Badge is behind the `fnod_badge` FeatureToggle

### Acceptance Criteria
- Badge renders if Veteran appellant is deceased and date of death is populated on appeal

### Testing Plan
1. In the `veterans` table, add a date of death string e.g. `2019-03-17` to a veteran file number like 500000000
3. Go to http://localhost:3000/queue
4. Search for the same veteran file number 
5. Verify that the FNOD badge appears in the Case List Table results
6. Click on the file number
7. Verify the badge appears on the Case Details page

### User Facing Changes

 BEFORE|AFTER
 ---|---
<> | <img width="1263" alt="Screen Shot 2020-12-17 at 1 47 50 PM" src="https://user-images.githubusercontent.com/68879427/102548122-0a1aad00-406f-11eb-9127-258818cded49.png">
<> | <img width="1288" alt="Screen Shot 2020-12-17 at 1 51 31 PM" src="https://user-images.githubusercontent.com/68879427/102548153-143cab80-406f-11eb-95be-be555c33af3e.png">


### Storybook Story
*For Frontend (Presentational) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the FnodBadge.jsx file
<img width="608" alt="Screen Shot 2020-12-17 at 2 02 21 PM" src="https://user-images.githubusercontent.com/68879427/102549158-8366cf80-4070-11eb-8b43-ebf78207b48b.png">
